dapper/mods/integration.py

```diff
@@ -10,8 +10,29 @@
 
 
 # fmt: off
-def rk4(f, x, t, dt, order=4):
-    """Runge-Kutta (explicit, non-adaptive) numerical ODE solvers.
+def rk4(f, x, t, dt, stages=4, s=0):
+    """Runge-Kutta (explicit, non-adaptive) numerical (S)ODE solvers.
+
+    This is a routine describing the standard 1-4 stage Runge-Kutta scheme with
+    uniform time steps.  The solver has the following number of stages / order
+    of convergence in deterministic / stochastic configurations.
+
+    | Stages | ODE Convergence | SDE Weak Convergence | SDE Strong Convergence |
+    ----------------------------------------------------------------------------
+    |      1 |            1.0  |                  1.0 |                    1.0 |
+    ----------------------------------------------------------------------------
+    |      2 |            2.0  |                  N/A |                    N/A |
+    ----------------------------------------------------------------------------
+    |      3 |            3.0  |                  N/A |                    N/A |
+    ----------------------------------------------------------------------------
+    |      4 |            4.0  |                  1.0 |                    1.0 |
+    ----------------------------------------------------------------------------
+
+    For stages=1, this becomes the Euler(-Maruyama) schemefor SDEs (s > 0.0)
+    with strong / weak convergence order 1.0 for SDEs with additive noise, see
+    `bib.grudzien2020numerical` and references therein for a general discussion
+    of numerical integegration schemes and discretization error analysis in the
+    DA cycle.
```

Comment:
I really do like tables, but I think in this case it's not needed. Also the reduction to Euler's scheme is mentioned under the `stages` keyword (below).
```suggestion
    """Runge-Kutta (explicit, non-adaptive) numerical (S)ODE solvers.

    For ODEs, the order of convergence equals the number of `stages`.
    For SDEs with additive noise (`s>0`), the order of convergence
    (both weak and strong) is 1 for any number of `stages`.
    See `bib.grudzien2020numerical` for a DA-specific discussion on
    integration schemes and their discretization errors.
```